python/ngen_cal/src/ngen/cal/ngen_hooks/ngen_output.py

```diff
@@ -0,0 +1,83 @@
+from ngen.cal import hookimpl
```

Comment:
This should fix this test failures!

```suggestion
from __future__ import annotations
```

Reply:
fixed in 082e2712272f339a94c5d01fa8a99f33a553b1ea